damus/damus.entitlements

```diff
@@ -7,6 +7,7 @@
 	<key>com.apple.developer.associated-domains</key>
 	<array>
 		<string>applinks:damus.io</string>
+		<string>webcredentials:damus.io</string>
```

Comment:
@jb55 You would need to add to the existing damus.io/.well-known/apple-app-site-association file to include the following on the same root level as `applinks`
```json
   "webcredentials": {
      "apps": [ "XK7H4JAB3D.com.jb55.damus2" ]
   },
```